src/main/kotlin/com/terraformation/backend/accelerator/db/ModuleEventStore.kt

```diff
@@ -33,32 +35,30 @@ class ModuleEventStore(
     private val eventsDao: EventsDao,
 ) {
   fun fetchOneById(eventId: EventId): EventModel {
-    requirePermissions { readModuleEventParticipants() }
-    val projectsField = eventProjectsMultiset()
-    return with(EVENTS) {
-      dslContext.select(asterisk(), projectsField).from(this).where(ID.eq(eventId)).fetchOne {
-        EventModel.of(it, projectsField)
-      } ?: throw EventNotFoundException(eventId)
-    }
+    requirePermissions { readModuleEvent(eventId) }
+    return fetchById(eventId = eventId).firstOrNull() ?: throw EventNotFoundException(eventId)
   }
 
-  fun fetchOneForProjectById(eventId: EventId, projectId: ProjectId): EventModel {
-    requirePermissions {
-      readModuleEvent(eventId)
-      readProject(projectId)
-    }
-
-    return with(EVENTS) {
-      dslContext
-          .selectFrom(this)
-          .where(ID.eq(eventId))
-          .and(
-              EVENTS.ID.`in`(
-                  DSL.select(EVENT_PROJECTS.EVENT_ID)
+  fun fetchById(
```

Comment:
We added this fetch function to replace fetching events as a multiset as part of a module